test_data_loader.py

```
@@ -0,0 +1,15 @@
+import pytest
+from data_loader import MotorImageryDataset
+
+def test_data_loader():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_data_loader`

Add return type annotation: `None`

(ANN201)